.github/workflows/build.yaml

```diff
@@ -27,7 +27,7 @@ jobs:
     - if: github.event_name == 'pull_request'
       run: npm run build:prefixed
       env:
-        BRANCH: ${{ github.head_ref }}
+        GATSBY_BRANCH: ${{ github.head_ref }}
```

Comment:
Was this a clarity thing, or was there a deeper reason it needed to happen?

Reply:
This was so we could expose this variable to client side code. Any env variable prefixed with `GATSBY_` becomes available to the client side. I wanted to be able to get the branch name for fixing the nav links for the preview. Used on [this line](https://github.com/keep-network/website/blob/3aef7775c2e10dc094fd9b945d1f202fae7cf252/src/components/NavScrollItem.js#L55)...Now looking back at this I'm thinking maybe I could have/should have used `withPrefix`. 🤦

Reply:
Interesting on both fronts (the fact that those env variables are passed along and the fact that we could have used `withPrefix` heh). All sounds good, will keep an eye out for what path you end up going down as the nav link bits get figured out.